BD-4448: Saving drafts for active campaigns GA (#9399)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/campaigns/managing_campaigns/change_your_campaign_after_launch.md

## Stopping your campaign

> To stop a campaign, open your **Campaign Details** page and select the **Stop Campaign** button in the bottom right of the page. When a campaign is stopped:

Reviewer: ```suggestion
To stop a campaign, open your **Campaign Details** page and select the **Stop Campaign**. When a campaign is stopped:
```

- Messages scheduled to be sent will be canceled.
- A/B tests where the initial test has already been sent will be permanently canceled.
- Events for messages that have already been sent (for example, open clicks) will still be tracked.

> - Campaigns can be restarted by selecting **Resume**.

Reviewer: ```suggestion
```

> After resuming, this campaign will continue sending messages and A/B tests, but any missed messages will not be re-sent or re-scheduled.

Reviewer: ```suggestion
To restart your campaign, select **Resume**. Your campaign will continue sending messages and A/B tests, but any missed messages will not be re-sent or re-scheduled.
```

{% alert important %}
This resets eligibility for people who already received the original campaign, so you may need to filter the duplicate campaign for people who did not receive the original.
{% endalert %}

## Saving drafts of active campaigns {#campaign-drafts}

> As you create and launch campaigns, you can also make edits to an active campaign and save it as a draft, allowing you to pilot your changes prior to another launch.

Reviewer: ```suggestion
Drafts are great for making large-scale changes to active campaigns. By creating a draft, you're able to pilot planned changes before your next launch.
```

> If you have a campaign that requires large-scale changes, you can create a draft for these edits.

Reviewer: ```suggestion
```

![A draft of an active campaign with an option to view the active campaign.]({% image_buster /assets/img/campaign_draft.png %})

> You can create drafts within an active campaign to build, save, and quality-check prior to launching these changes in the active campaign. These drafts don't have any analytics since the draft changes aren't launched yet. A campaign can only have one draft at a time.

Reviewer: ```suggestion
{% alert note %}
A campaign can only have one draft at a time. Additionally, analytics aren't available since drafted changes aren't launched yet.
{% endalert %}
```

To create a draft, do the following:

1. Go to your active campaign.
2. Make your changes.

> 3. Select the **Save as Draft** button in the campaign footer. Note that edits to the active campaign cannot be made while a draft of a campaign exists. You can update the campaign to apply changes, or discard the draft.

Reviewer: ```suggestion
3. Select **Save as Draft**. Note that after creating a draft, you cannot make edits to the active campaign until you either launch or discard your draft.
```

> To reference the active campaign, select **View Active Campaign** in the footer from the analytics view or the campaign header from the draft. To return to an active campaign, select **Edit Draft** from the analytics view or the active campaign view.

Reviewer: ```suggestion
You can reference the active campaign in the header of the campaign draft, or in the footer of the campaign analytics after selecting **View Active Campaign**. To return to an active campaign, select **Edit Draft** from the analytics view or the active campaign view.
```

not a perfect rewrite, but trying to find a way to make it more clear. also what is meant by "reference" here?
